hasura/migrations/carnet_de_bord/1674050275074_change_orientation_type_to_system/up.sql

```diff
@@ -0,0 +1,81 @@
+
+-- notebook_info table
+
+alter table "public"."notebook_info" drop constraint "notebook_info_orientation_fkey";
+
+alter table "public"."notebook_info" add column "orientation_system_id" uuid
+ null;
+
+alter table "public"."notebook_info"
+  add constraint "notebook_info_orientation_system_id_fkey"
+  foreign key ("orientation_system_id")
+  references "public"."orientation_system"
+  ("id") on update cascade on delete cascade;
+
+alter table "public"."notebook_info" drop constraint "notebook_info_orientation_system_id_fkey",
+  add constraint "notebook_info_orientation_system_id_fkey"
+  foreign key ("orientation_system_id")
+  references "public"."orientation_system"
+  ("id") on update set null on delete set null;
+
+alter table "public"."notebook_info" drop constraint "notebook_info_orientation_system_id_fkey",
+  add constraint "notebook_info_orientation_system_id_fkey"
+  foreign key ("orientation_system_id")
+  references "public"."orientation_system"
+  ("id") on update cascade on delete cascade;
+
+WITH subquery AS (
+    SELECT id, orientation_type
+    FROM orientation_system
+    WHERE name IN ('Pro','Social','Socio-pro')
+)
+UPDATE notebook_info
+SET orientation_system_id=subquery.id
+FROM subquery
+WHERE notebook_info.orientation=subquery.orientation_type;
```

Comment:
Cette requête (ainsi que celles similaires qui suivent sur `orientation_request`) peut être légèrement simplifiée en :
```suggestion
UPDATE notebook_info
SET orientation_system_id=orientation_system.id
FROM orientation_system
WHERE orientation_system.name IN ('Pro','Social','Socio-pro')
  AND notebook_info.orientation=orientation_system.orientation_type;
```